packages/firestore/src/remote/datastore.ts

```diff
@@ -246,20 +249,21 @@ export async function invokeRunAggregationQueryRpc(
   aggregates: Aggregate[]
 ): Promise<ApiClientObjectMap<Value>> {
   const datastoreImpl = debugCast(datastore, DatastoreImpl);
+  const target = queryToTarget(query);
   const { request, aliasMap } = toRunAggregationQueryRequest(
     datastoreImpl.serializer,
-    queryToTarget(query),
+    target,
     aggregates
   );
+  const encodedPath = toQueryTargetPath(datastoreImpl.serializer, target, true);
```

Comment:
This is the second instance of the fundamental change in this PR. See my other notes above.